payment/src/main/java/org/killbill/billing/payment/core/sm/payments/PaymentOperation.java

```diff
@@ -70,7 +70,7 @@ protected PaymentOperation(final GlobalLocker locker,
     public OperationResult doOperationCallback() throws OperationException {
         final String pluginName;
         try {
-            pluginName = daoHelper.getPaymentProviderPluginName();
+            pluginName = daoHelper.getPaymentProviderPluginName(true);
             this.plugin = daoHelper.getPaymentPluginApi();
```

Comment:
Same remark as before: we only want to fetch deleted payment method for GET operations, then shouldn't this be false? By the way interestingly enough we end up doing twice the call to `getPaymentProviderPluginName`  (once directly and once through `getPaymentPluginApi`). Is there a possibility simplify that (if it's too complicated drop it).